CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_pendo v0.4.0
+## 🚨 Breaking Changes 🚨:
```

Comment:
would the changes in this package be a breaking change?

Reply:
Note for the history--this is a breaking change because of changes in the source.